android/build.gradle

```diff
@@ -1,23 +1,24 @@
 
 buildscript {
     repositories {
+        google()
```

Comment:
did you tested, this is building fine on android?

Reply:
Hi. Sure thing. Yes, I tried it on one of my projects and it builds fine. I can't see what causes the pipeline to fail.